src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx

```diff
@@ -39,7 +39,7 @@ export const CephFs: VolumeVariantComponent<"cephfs"> = (
             </DrawerItem>
           )
       }
-      <DrawerItem name="Readonly">
+      <DrawerItem name="Readonly" valueTestId="cephfs-readonly">
```

Comment:
Why to use extra `props.valueTestId` when we can pass directly `<Drawer data-testid="some-value"/>` with full `...props` or _(limited/filtered props)_ to `<span>`?

Reply:
I don't see where we can pass in `data-testid` on `Drawer`? That only works on the base HTML react elements. Plus we aren't producing a `Drawer` anyway.

Reply:
Nope, it works on all elements and react components. You should pass it on `DrawerItem` not `Drawer` _(wrong example above)_

Reply:
Nope, it works on all elements and react components. You should pass it on `DrawerItem` not `Drawer` _(wrong example above)_

Reply:
For auto-complete you can declare in `DrawerItem`'s props, but it's not required.

Reply:
I agree that unusual `valueTestId` should be avoided.

Reply:
Oh thanks for the info. I have now done that.